relate/src/main/scala/com/lucidchart/relate/SqlRow.scala

```diff
@@ -73,8 +73,8 @@ class SqlRow(val resultSet: java.sql.ResultSet) extends ResultSetWrapper {
   def strictShortOption(column: String): Option[Short] = getResultSetOption(resultSet.getShort(column))
   def strictSQLXML(column: String): SQLXML = resultSet.getSQLXML(column)
   def strictSQLXMLOption(column: String): Option[SQLXML] = getResultSetOption(resultSet.getSQLXML(column))
-  def strictString(column: String): String = resultSet.getString(column)
-  def strictStringOption(column: String): Option[String] = getResultSetOption(resultSet.getString(column))
+  def strictString(column: String): String = string(column)
```

Comment:
We are changing strictString to get the "throw NoSuchElementException` behavior, rather than `null` or `0`. Why aren't we changing strictLong as well? (Again, this is why I'd 'flip' the implementation of strictString and strictStringOption back)

Reply:
Oh, I meant to undo that change.